### DISCUSSION TITLE
Buying a new Asus notebook and want to know if G-Helper is fully supported?

### DISCUSSION THREAD

**Discussion by 205g0** [seerge/g-helper#1492] [Polls]
I still don't know the indicators for G-Helper to work. Does it have to be an notebook from the ROG series? No, because some Vivobooks seem to be supported as well. And is every feature supported? I think it would be great to have something to know this before without doing tons of research

**Comment by seerge** [comment]
@205g0 hello, all modern asus laptops are "supported" :) if certain feature (like disabling dGPU) doesn't exist on your device - app just won't show it :)

**Reply by 205g0** [reply]
Wow, really, this is amazing, so also non-ROG ones such as the Zenbook S 13?! "Modern" means from some specific date or year?
